index.html

```diff
@@ -169,5 +169,25 @@ <h2 class="more_det">LET’S CONNECT</h2>
              </ul>
          </div>
          </section>
+         <section class="form-bui">
+            <h3 class="my-contact">Contact me</h3>
+            <p class="my_info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/xeqvdqoj">
+                <input type="text" name="user_name" id="name" maxlength="30" placeholder="Yeremias" required>
+                <input type="email" name="email" id="email" placeholder="natajayanj14@gmail.com" required>
+                <textarea name="message" id="message-form" rows="5">Write your message here</textarea>
+            </form>
+            <ul class="get-touch">
+                <li class="get-tou">GIt Touch</li>
```

Comment:
- [ ] Replace ul and li tag with a button with a type="submit" property. This is because the form needs a button to submit the data.
- [ ] This button should be within the form tag.
- [ ] The button text should say "Get It Touch".
- [ ] Kindly style the button for active and hover state.
![Captura de pantalla_2021-09-28_21-36-08](https://user-images.githubusercontent.com/53111464/135193715-23744fe7-b6eb-4b7a-83e1-610ac32e2781.png)
- [ ] The form is not sending data kindly fix it.